### DISCUSSION TITLE
Markdown links repetition

### DISCUSSION THREAD

**Discussion by peterpeterparker** [facebook/docusaurus#8595] [Q&A]
Hi, I'm pretty new to Docusaurus. Is there a way in markdown to avoid repeating markdown links?

e.g. if I have to use the link `[something](./docs/otherfolder/something.md)` in multiple pages, is there a way to declare it once  and use on `[something]` in all pages?  

I am aware it's possible in a single markdown page - specifying `[something]: ./docs/otherfolder/something.md` at the bottom of a file - but not sure how to solve this globally.

**Comment by slorber** [comment]
`[something]` is not a std markdown syntax and we try to not diverge too much by default. If you want that you should invent your own syntax by creating a remark plugin.

My suggestion, wait for MDX 2, it will make it much easier to write a [remark directive](https://github.com/remarkjs/remark-directive) plugin.

For example, a quite simple remark plugin you could turn:
- `click :something` to `click [something](./docs/otherfolder/something.md)`
- `click :something[there]` to `click [there](./docs/otherfolder/something.md)`

It's possible that we provide such a plugin out of the box in the future but for now it's your responsibility to create it

**Reply by peterpeterparker** [reply]
Gotcha, thanks for the feedback
